resources/assets/uad_lists.json

```diff
@@ -16832,11 +17309,11 @@
   {
     "id": "com.google.ar.core",
     "list": "Google",
-    "description": "YouTube app (https://play.google.com/store/apps/details?id=com.google.android.youtube)\nGoogle Play Services for AR (Augmented Reality) (https://play.google.com/store/apps/details?id=com.google.ar.core)\n",
+    "description": "Google Play Services for AR (Augmented Reality) (https://play.google.com/store/apps/details?id=com.google.ar.core)\nDisabling breaks some games that depend on it, like Pokemon GO.",
     "dependencies": null,
     "neededBy": null,
     "labels": null,
-    "removal": "Recommended"
+    "removal": "Advanced"
```

Comment:
I suppose it is pretty easy to reinstall/enable, the only issue I see is that you don't get a clear error message; the game just doesn't work. So users that do get rid of the package might later have a hard time finding the issue.
The package also only runs on its own at boot, so it's not like it's always active.

Reply:
Runs on boot = runs for a few seconds on boot.
**It doesn't keep running in the background**.

Reply:
> Runs on boot = runs for a few seconds on boot.

Strictly speaking, as per the docs:

> Allows an application to receive the [Intent.ACTION_BOOT_COMPLETED](https://developer.android.com/reference/android/content/Intent#ACTION_BOOT_COMPLETED) that is broadcast after the system finishes booting. [...] it can have a negative impact on the user experience by increasing the amount of time it takes the system to start **and allowing applications to have themselves running without the user being aware of them**.
[Source](https://developer.android.com/reference/android/Manifest.permission#RECEIVE_BOOT_COMPLETED)

So maybe this specific app only run few seconds at boot but it can be used by apps to automatically run in the background as soon as the phone starts up.

Reply:
> it can be used by apps to automatically run in the background as soon as the phone starts up.

I know the permissions/broadcast receivers *could* mean that, but I'm not using terminology linked to them.
"Runs on boot" is my own term, which I've clarified in a couple of descriptions to mean that it only runs for a few sec and doesn't run in the background after boot.
**I gathered the data on whether packages "Run on boot" by *testing***, so if I say a package runs in the background or on boot it's because **it did on my phone**.

Reply:
After testing a bit more, it apparently only breaks Pokemon GO in Android 9, the game runs fine without it in Android 11.
Let's just go with Recommended then.